doc/rst/language/spec/generics.rst

```diff
@@ -897,6 +907,40 @@ argument.
    defines a two-element list with nodes containing the values ``1`` and
    ``2``. The type of each object could be specified as ``Node(int)``.
 
+ .. _Fields_with_Generic_Types:
+
+Fields with Generic Types
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A field declared with a generic type is similar to a field declared
+without any type (:ref:`Fields_without_Types`); however, the generic type
+constrains the possible field types to instantiations of the generic.
+The types for such fields must either be a built-in generic type (see
+:ref:`Built_in_Generic_Types`) or include a ``?`` to mark them as generic
+(see :ref:`Marking_Generic_Types`).
+
+   *Example (fieldWithGenericType.chpl)*.
+
+   This code defines a generic record ``queue`` and then a generic record
+   ``wrapper`` that uses a field to hold a.
```

Comment:
I think you forgot a word or two at the end of this sentence.  Probably ``queue``?  Or maybe "generic ``queue``"?